examples/DQN/atari_model.py

```diff
@@ -0,0 +1,67 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+import parl.layers as layers
+from parl.framework.model_base import Model
+from parl.utils import logger
+
+
+class AtariModel(Model):
+    def __init__(self, img_height, img_width, act_dim):
+        self.img_height = img_height
+        self.img_width = img_width
+        self.act_dim = act_dim
+
+        self.conv1 = layers.conv2d(
+            num_filters=32,
+            filter_size=5,
+            stride=[1, 1],
```

Comment:
Please use `integer` type as that in `filter_size`, I have check that in 
https://github.com/PaddlePaddle/Paddle/blob/6d04a9cf4779a63123c5c1254450dbf881961b1f/python/paddle/fluid/layers/nn.py#L1694
It also support passing an integer to `stride` and `padding`